static/scripts/PlotHandler.js

```diff
@@ -0,0 +1,121 @@
+import { addNewPlotEntry } from "./workspace";
+
+class PlotHandler {
+    constructor() {
+        this.plotMap = new Map();
+    }
+    //the data forwarded contains the information: data(yValue,datasetName,plotName,plotType)
```

Comment:
is it possible to use an object instead of an tuple? I think it would be clearer if data would be an object of the type `{yValue, datasetName, plotName, plotType}`